### PR TITLE
Add Debugging Chrome Extensions video

### PR DESCRIPTION
**Pull request by jecfish**

### CODE REVIEW

File: site/en/docs/extensions/mv3/getstarted/extensions-101/index.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ tutorials do not include a popup.
 
 ## Developing your extension {: #development }
 
+{% YouTube id='kVwX3Ci1SxU' %}
```
Comment (oliverdunk) [inline site/en/docs/extensions/mv3/getstarted/extensions-101/index.md:91]: Could we move this to the bottom of the section, maybe with a new subheading "Debugging", that also links to the debugging page?

Comment (jecfish) [reply site/en/docs/extensions/mv3/getstarted/extensions-101/index.md]: Good point, movend and added a new debugging section. Over to @jpmedley to review the writing & decide where to place the section

**Comment by jecfish** [comment]
Do not merge this PR until DevTools Tips video went live

File: site/en/docs/extensions/mv3/getstarted/extensions-101/index.md (hunk 1 above)
Comment (oliverdunk) [reply site/en/docs/extensions/mv3/getstarted/extensions-101/index.md]: Note from Joe: Can we move this video below the paragraph of text?

**Review by oliverdunk — APPROVED** [review]
LGTM

**Comment by jecfish** [comment]
@oliverdunk please help to update the new youtube url when it's ready.

File: site/en/docs/extensions/mv3/getstarted/extensions-101/index.md [hunk 2]
```diff
@@ -92,6 +92,12 @@ Even though web applications and extensions share many of the same technologies,
 experience is different. Check out [Development Basics][doc-dev-basics] to create a "Hello,
 Extensions" example and familiarize yourself with the extension development workflow.
 
+## Debugging your extension {: #debugging }
+
+Extensions can access the same Chrome DevTools as web pages. To become an expert in debugging extensions, you will need to know how to locate logs and errors of the different extension components. Check out [Debugging extensions](/docs/extensions/mv3/tut_debugging/) to learn thes fundamental techniques for debugging your extension.
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/getstarted/extensions-101/index.md:97]:
```suggestion
Extensions can access the same Chrome DevTools as web pages. To become an expert in debugging extensions, you will need to know how to locate logs and errors of the different extension components. Check out [Debugging extensions](/docs/extensions/mv3/tut_debugging/) to learn techniques for debugging your extension.
```

File: site/en/docs/extensions/mv3/tut_debugging/index.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ updated: 2023-08-09
 description: Instructions for debugging Chrome Extensions.
 ---
 
+{% YouTube id='Ta-YTDhiBIQ' %}
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/tut_debugging/index.md:10]: Videos shouldn't be introduced without context. please put this after the introductory paragraph. While you're here, can you please remove the 'Overview' heading for us? It's redundant and we're trying to get rid of them generally.

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**